travis_demo/nicole_task.py

```diff
@@ -0,0 +1,2 @@
+def ifEvenOdd(n):
```

Comment:
Typically, function names are lower-cased with `_`'s separating words, in python. Can you change this?